src/hexen/p_setup.c

```diff
@@ -806,6 +806,12 @@ void P_SetupLevel(int episode, int map, int playermask, skill_t skill)
     {                           // Probably fog ... don't use fullbright sprites
         LevelUseFullBright = false;
     }
+#ifdef CRISPY_TRUECOLOR
+    // [cirspy] TODO - two problems:
+    // 1) Need to re-generate colormaps, otherwise brightest light levels will be screwed up.
+    // 2) Foggy maps, or technically, ANY map with custom colormap MUST use pregenerated colormap lump.
+    R_InitColormaps();
```

Comment:
Possible approach: query custom colormap as `byte`, not as `int` as [currently done](https://github.com/fabiangreffrath/crispy-doom/blob/master/src/hexen/p_setup.c#L71) in the code, so we can use given name for processing it like done [here](https://github.com/fabiangreffrath/crispy-doom/blob/master/src/heretic/r_data.c#L561). Unoptimal since what will happen if `F11` will be pressed on level with such colormap? Yep, it will be rewritten with standard one.

Another approach: say "screw you" to the mods and always `COLORMAP` and `FOGMAP` tables. Much easier but not acceptable.